src/mlpack/bindings/python/print_doc_functions.hpp

```diff
@@ -52,26 +52,29 @@ inline std::string PrintValue(const bool& value, bool quotes);
 /**
  * Given a parameter name, print its corresponding default value.
  */
-inline std::string PrintDefault(const std::string& paramName);
+inline std::string PrintDefault(const std::string& bindingName,
+																const std::string& paramName);
```

Comment:
```suggestion
                                const std::string& paramName);
```
Not sure what happened here. :)

Reply:
Sorry for so many style issues :(, surprisingly they do not show up in my editor. I use VSCode, maybe it is time to change it and use something else.

Reply:
No worries!  Maybe it is a spaces vs. tabs setting or something.